docs/src/main/paradox/scala/http/routing-dsl/directives/future-directives/onComplete.md

```diff
@@ -15,4 +15,12 @@ To complete with a successful result automatically and just handle the failure r
 
 ## Example
 
-@@snip [FutureDirectivesExamplesSpec.scala](../../../../../../../test/scala/docs/http/scaladsl/server/directives/FutureDirectivesExamplesSpec.scala) { #onComplete }
\ No newline at end of file
+@@snip [FutureDirectivesExamplesSpec.scala]($test$/scala/docs/http/scaladsl/server/directives/FutureDirectivesExamplesSpec.scala) { #onComplete }
```

Comment:
I changed the paths @raboof - should be good to merge now I guess?

Reply:
:+1: if jenkins agrees :)